frontend/src/app/main/ui/workspace/sidebar/options/menus/color_selection.cljs

```diff
@@ -56,8 +58,13 @@
         groups-ref       (h/use-ref-value groups)
         prev-colors-ref  (mf/use-ref nil)
 
-        ;; grouped-colors*  (mf/use-var nil)
-        ;; prev-colors*     (mf/use-var [])
+        color-keys*
+        (mf/use-var
+         (->> (concat colors library-colors)
+              (reduce
+               (fn [result color]
+                 (assoc result color (dm/str (uuid/next))))
+               {})))
```

Comment:
The var, in the same way as use-ref, should be initialized on an effect if the value is not primitive. If not, this concat and reduce will be executed all the time the component is rendered and the result will be ignored (with the exception of first render)